cmsplugin_filer_image/migrations/0002_delete_duplicate_thumbnailoptions.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from django.db.models import Count, Max
+
+
+def forward(apps, schema_editor):
+    """
+    Get all the duplicates for ThumbnailOption.
+    Point each FilerImage where the duplicates are used to the correct
+    ThumbnailOption.
+    Delete remaining ThumbnailOption duplicates.
+    """
+    ThumbnailOption = apps.get_model("cmsplugin_filer_image", "ThumbnailOption")
+    unique_fields = ["name", "width", "height", "crop", "upscale"]
+    duplicates = (ThumbnailOption.objects.values(*unique_fields)
```

Comment:
Since you are annotating the max_id you should add it to the list of values otherwise I think it will execute other queries for each duplicate['max_id'].